modules/linear_boltzmann_solvers/executors/pi_keigen_scdsa.cc

```diff
@@ -79,93 +79,12 @@ XXPowerIterationKEigenSCDSA::XXPowerIterationKEigenSCDSA(const InputParameters&
-  // uk_man.AddUnknown(UnknownType::VECTOR_N, num_gs_groups);
-  //
-  //// Make boundary conditions
-  // auto bcs = TranslateBCs(lbs_solver_.SweepBoundaries(),
-  //                                       /*vaccum_bcs_are_dirichlet=*/true);
-  //
-  //// Make xs map
-  // auto matid_2_mgxs_map =
-  //   PackGroupsetXS(lbs_solver_.GetMatID2XSMap(),
-  //                                front_gs_.groups_.front().id_,
-  //                                front_gs_.groups_.back().id_);
-  //
-  //// Create solver
-  // const auto& sdm = lbs_solver_.SpatialDiscretization();
-  // const auto& unit_cell_matrices = lbs_solver_.GetUnitCellMatrices();
-  //
-  // if (diffusion_solver_sdm_ == "pwld")
-  //   diffusion_solver_ = std::make_shared<DiffusionMIPSolver>(
-  //     std::string(TextName() + "_WGDSA"),
-  //     sdm,
-  //     uk_man,
-  //     bcs,
-  //     matid_2_mgxs_map,
-  //     unit_cell_matrices,
-  //     true); // verbosity
-  // else
-  //{
-  //   continuous_sdm_ptr_ =
-  //     SpatialDiscretization_PWLC::New(sdm.ref_grid_);
-  //   diffusion_solver_ = std::make_shared<DiffusionPWLCSolver>(
-  //     std::string(TextName() + "_WGDSA"),
-  //     *continuous_sdm_ptr_,
-  //     uk_man,
-  //     bcs,
-  //     matid_2_mgxs_map,
-  //     unit_cell_matrices,
-  //     true); // verbosity
-  //   requires_ghosts_ = true;
-  //   lbs_pwld_ghost_info_ = MakePWLDVecGhostCommInfo(
-  //     lbs_solver_.SpatialDiscretization(), lbs_solver_.UnknownManager());
-  //
-  //   const auto& cfem_sdm = *continuous_sdm_ptr_;
-  //   const auto ghost_dof_ids =
-  //     cfem_sdm.GetGhostDOFIndices(lbs_solver_.UnknownManager());
-  // }
-  //
-  //{
-  //   typedef const std::string cstr;
-  //   cstr l_abs_tol = "diff_accel_diffusion_l_abs_tol";
-  //   cstr max_iters = "diff_accel_diffusion_max_iters";
-  //   cstr verbose = "diff_accel_diffusion_verbose";
-  //   cstr petsc_options = "diff_accel_diffusion_petsc_options";
-  //
-  //   auto& ds = diffusion_solver_;
-  //
-  //   ds->options.residual_tolerance = params.GetParamValue<double>(l_abs_tol);
-  //   ds->options.max_iters = params.GetParamValue<int>(max_iters);
-  //   ds->options.verbose = params.GetParamValue<bool>(verbose);
-  //   if (not params.GetParamValue<std::string>(petsc_options).empty())
-  //     ds->options.additional_options_string =
-  //       params.GetParamValue<std::string>(petsc_options);
-  // }
-  //
-  // log.Log() << "Initializing diffusion solver";
-  // diffusion_solver_->Initialize();
-  //  opensn::mpi_comm.barrier();
-  // log.Log() << "Done Initializing diffusion solver";
-  //
-  // log.Log() << "Assembling A and b";
-  // std::vector<double> dummy_rhs;
-  // if (diffusion_solver_sdm_ == "pwld")
-  //   dummy_rhs.assign(sdm.GetNumLocalDOFs(uk_man), 0.0);
-  // else
-  //   dummy_rhs.assign(continuous_sdm_ptr_->GetNumLocalAndGhostDOFs(uk_man),
-  //   0.0);
-  //
-  // diffusion_solver_->AssembleAand_b(dummy_rhs);
-  // log.Log() << "Done Assembling A and b";
```

Comment:
:+1:

Reply:
what was this about? Is DFEM still available with scdsa ??

Reply:
It looks like this is in `Initialize`. I'm not sure why it was ever in the constructor, to be honest.